HyprMXAdapter/src/main/java/com/chartboost/mediation/hyprmxadapter/HyprMXAdapter.kt

```diff
@@ -385,18 +391,23 @@ class HyprMXAdapter : PartnerAdapter {
     ): Result<PartnerAd> {
         PartnerLogController.log(LOAD_STARTED)
 
-        return when (request.format.key) {
-            AdFormat.BANNER.key, "adaptive_banner" -> loadBannerAd(
-                context,
-                request,
-                partnerAdListener
-            )
+        if (loadedPartnerPlacements.contains(request.partnerPlacement)) {
```

Comment:
while this is fine, I would encourage using if -> return
and then leaving the rest of the code alone (so you don't have the else statement and the indent).
I say that as I'm going to request you bring this change into the `feature/vc/mediation-5.0.0` branch and it'll be a lot easier to apply this commit if you don't touch the stuff below this `if` check ;)

Reply:
Ok, no problem.  Thanks for the suggestion.

Reply:
I've taken care of this suggestion.  But I'm curious if you can think of a better idea than what I've done in this PR?  I'm concerned that others will still think there is a bug with the adapter.  Currently, this path has an error with the final show (the reported error in the JIRA ticket).

LOAD -> LOAD -> SHOW -> SHOW

However, this change still has an error, now moved to the second load, but a different kind of error:

LOAD -> LOAD -> SHOW

Ideally, we need a way to prevent the ad queue from even trying to put another ad in the queue for this adapter. Then things will be error free.